doc/examples/viz_advanced.py

```diff
@@ -0,0 +1,187 @@
+module was named ``fvtk``, meaning functions using vtk. This is still available
+for backwards compatibility but now there is a more comprehensive way to access
+the main functions using the following modules.
+"""
+
+import numpy as np
+from dipy.viz import actor, window, widget
+
+"""
+In ``window`` we have all the objects that connect what needs to be rendered
+to the display or the disk e.g. for saving screenshots. So, there you will find
+key objects and functions like the ``Renderer`` class which holds and provides
+access to all the actors and the ``show`` function which displays what is
+in the renderer on a window. Also, this module provides access to functions
+for opening/saving dialogs and printing screenshots (see ``snapshot``).
+
+In the ``actor`` module we can find all the different primitives e.g.
+streamtubes, lines, image slices etc.
+
+In the ``widget`` we have some other objects which allow to add buttons
+and sliders and these interact both with windows and actors. Because of this
+they need input from the operating system so they can process events.
+
+So, let's get started. In this tutorial, we will visualize some bundles
+together with FA or T1. We will be able to change the slices using
+a ``slider`` widget.
+
+First we need to fetch and load some datasets.
+"""
+
+from dipy.data.fetcher import fetch_bundles_2_subjects, read_bundles_2_subjects
+
+fetch_bundles_2_subjects()
+
+"""
+The following function outputs a dictionary with the required bundles e.g. af
+left and maps, e.g. FA for a specific subject.
+"""
+
+res = read_bundles_2_subjects('subj_1', ['t1', 'fa'],
+                              ['af.left', 'cst.right', 'cc_1'])
+
+"""
+We will use 3 bundles, FA and the affine transformation that brings the voxel
+coordinates to world coordinates (RAS 1mm).
+"""
+
+streamlines = res['af.left'] + res['cst.right'] + res['cc_1']
+data = res['fa']
+shape = data.shape
+affine = res['affine']
+
+"""
+With our current design it is easy to decide in which space you want the
+streamlines and slices to appear. The default we have here is to appear in
+world coordinates (RAS 1mm).
+"""
+
+world_coords = True
+
+"""
+If we want to see the objects in native space we need to make sure that all
+objects which are currently in world coordinates are transformed back to
+native space using the inverse of the affine.
+"""
+
+if not world_coords:
+    from dipy.tracking.streamline import transform_streamlines
+    streamlines = transform_streamlines(streamlines, np.linalg.inv(affine))
+
+"""
+Now we create, a ``Renderer`` object and add the streamlines using the ``line``
+function and an image plane using the ``slice`` function.
+"""
+
+ren = window.Renderer()
+stream_actor = actor.line(streamlines)
+
+if not world_coords:
+    image_actor = actor.slicer(data, affine=np.eye(4))
+else:
+    image_actor = actor.slicer(data, affine)
+
+"""
+For fun let's change also the opacity of the slicer
```

Comment:
"For fun let's change also..." => "For demonstration purposes let's also change..."